great_expectations/render/renderer/inline_renderer.py

```diff
@@ -0,0 +1,127 @@
+    ) -> List[RenderedAtomicContent]:
+        """Gets RenderedAtomicContent for a given ExpectationConfiguration or ExpectationValidationResult.
+
+        Args:
+            render_object: The object to render
+
+        Returns:
+            A list of RenderedAtomicContent objects for a given ExpectationConfiguration or ExpectationValidationResult.
+        """
+        expectation_type: str
+        atomic_renderer_prefix: str
+        legacy_renderer_prefix: str
+        if isinstance(render_object, ExpectationConfiguration):
+            expectation_type = render_object.expectation_type
+            renderer_prefix = "atomic.prescriptive"
+        elif isinstance(render_object, ExpectationValidationResult):
+            expectation_type = render_object.expectation_config.expectation_type
+            renderer_prefix = "atomic.diagnostic"
+        else:
+            raise ValueError(
+                f"object must be of type ExpectationConfiguration or ExpectationValidationResult, but an object of type {type(render_object)} was passed"
+            )
+
+        renderer_names: List[str] = get_renderer_names_with_renderer_prefix(
+            object_name=expectation_type,
+            renderer_prefix=renderer_prefix,
+        )
+
+        rendered_content: List[
+            RenderedAtomicContent
+        ] = self._get_atomic_rendered_content_from_renderer_names(
+            render_object=render_object,
+            renderer_names=renderer_names,
+            expectation_type=expectation_type,
+        )
+
+        return rendered_content
+
+    def _get_atomic_rendered_content_from_renderer_names(
+        self,
+        render_object: Union[ExpectationConfiguration, ExpectationValidationResult],
+        renderer_names: List[str],
+        expectation_type: str,
+    ) -> List[RenderedAtomicContent]:
+        default_prescriptive_renderer_name: str = "atomic.prescriptive.kwargs"
+        rendered_content: List[RenderedAtomicContent] = []
+
+        for renderer_name in renderer_names:
+            if renderer_name != default_prescriptive_renderer_name:
+                try:
+                    renderer_rendered_content = (
+                        self._get_renderer_atomic_rendered_content(
+                            render_object=render_object,
+                            renderer_name=renderer_name,
+                            expectation_type=expectation_type,
+                        )
+                    )
+                    rendered_content.append(renderer_rendered_content)
+                except Exception:
+                    pass
+
+        if len(rendered_content) == 0 and isinstance(
+            render_object, ExpectationConfiguration
+        ):
+            renderer_rendered_content = (
+                InlineRenderer._get_renderer_atomic_rendered_content(
+                    render_object=render_object,
+                    renderer_name=default_prescriptive_renderer_name,
```

Comment:
Only use renderer `atomic.prescriptive.kwargs` if we are rendering an `ExpectationConfiguration` and no other atomic renderers existed/rendered successfully.